From: Mathias Gibbens <mathias@calenhad.com>
Description: Fix FTBFS on armel and armhf architectures by adding missing "armv8l" architecture. Based on upstream comment, listing with the 32bit armv7 architectures.
Forwarded: https://github.com/lxc/lxd/issues/10909
diff --git a/shared/osarch/architectures.go b/shared/osarch/architectures.go
index c75cc5e95..9bd695060 100644
--- a/shared/osarch/architectures.go
+++ b/shared/osarch/architectures.go
@@ -38,7 +38,7 @@ var architectureNames = map[int]string{
 var architectureAliases = map[int][]string{
 	ARCH_32BIT_INTEL_X86:             {"i386", "i586", "386", "x86", "generic_32"},
 	ARCH_64BIT_INTEL_X86:             {"amd64", "generic_64"},
-	ARCH_32BIT_ARMV7_LITTLE_ENDIAN:   {"armel", "armhf", "arm", "armhfp", "armv7a_hardfp", "armv7", "armv7a_vfpv3_hardfp"},
+	ARCH_32BIT_ARMV7_LITTLE_ENDIAN:   {"armel", "armhf", "arm", "armhfp", "armv7a_hardfp", "armv7", "armv7a_vfpv3_hardfp", "armv8l"},
 	ARCH_64BIT_ARMV8_LITTLE_ENDIAN:   {"arm64", "arm64_generic"},
 	ARCH_32BIT_POWERPC_BIG_ENDIAN:    {"powerpc"},
 	ARCH_64BIT_POWERPC_BIG_ENDIAN:    {"powerpc64", "ppc64"},
